src/Umbraco.Web.UI/umbraco/config/lang/es.xml

```diff
@@ -273,53 +273,53 @@
     <area alias="bulk">
         <key alias="done">Hecho</key>
 
-        <key alias="deletedItem">Borrado %0% elemento</key>
-        <key alias="deletedItems">Borrados %0% elementos</key>
-        <key alias="deletedItemOfItem">Borrado %0% de %1% elemento</key>
-        <key alias="deletedItemOfItems">Borrados %0% de %1% elementos</key>
+        <key alias="deleteditam">Borrado %0% elemento</key>

```

Comment:
Hi @gdiazderadaa ,

Many thanks for your PR. I can see that some of the key aliases has been changed. Can you revert it?

Regards,
Poornima, [Community Pull Request team](https://umbraco.com/blog/introducing-the-new-cpr-team/)

Reply:
done!